lang/strings_english.txt

```diff
@@ -1479,6 +1479,7 @@ $s_email_on_deleted = 'E-mail on Deleted';
 $s_email_on_sponsorship_changed = 'E-mail on Sponsorship changed';
 $s_email_on_relationship_changed = 'E-mail on Relationship changed';
 $s_email_on_updated = 'E-mail on Updated';
+$s_email_on_monitor = 'E-mail on Monitor Added';
```

Comment:
How about something like `Email on User Monitoring Changes`?  We can brainstorm on the appropriate text, but the one in the PR is not ideal.  Is it also only when a user monitors an issue, or is it also trigger when a user unmonitors issue?  Does it apply to cases where a user adds others?

Reply:
>  Is it also only when a user monitors an issue, or is it also trigger when a user unmonitors issue? Does it apply to cases where a user adds others?

Called from the function bug_monitor() in bug_api, so whenever a new monitoring user is added.
Not when it is removed

> How about something like Email on User Monitoring Changes?

You are better with the english descriptions.
Based on it's usage, is it better as `Email on User Monitoring Added`?

Reply:
"Email when someone starts monitoring"
"Email when user starts monitoring"

Reply:
I would personally prefer "Email on User Monitoring Changes" and make it trigger emails when `bug_monitor()` or `bug_unmonitor()` are called.  We should also update related documentation.

Reply:
> I would personally prefer "Email on User Monitoring Changes" and make it trigger emails when bug_monitor() or bug_unmonitor() are called. We should also update related documentation.

I didn't plan changing the underlying behavior. However, this may be the less useful notification type (and is all OFF by default), so that wouldn't have much impact.
But note that, in case of going that way, the email sent to the user that is added as monitor is not managed with this setting (it's sent as type "explicit"). Do you suggest to also email explicitly the target user when is removed from monitoring?
I'd say, to not change that, becasue unmonitoring usually means i don't want more notifications from that issue.

Reply:
> unmonitoring usually means i don't want more notifications from that issue.

True, but user wanting to be notified about unmonitoring event probably depends on whether it's the user's own action, or if another person (eg. admin) removed them from monitoring list...

Reply:
@vboctor I don't think we should change the system's behavior (i.e. send notifications on "unmonitoring") as part of this PR. If this is indeed a requirement (and I don't believe it is), then it should be tracked as a separate work item. Let's stick to the PR's current scope for now. 

Considering that there was no update on the label discussion for nearly a month, do you agree to keep @cproensa's original wording, or would you prefer one of my suggestions ?

My favorite is "Email when user starts monitoring"

Reply:
I have updated with @dregad suggestion: `E-mail when user starts monitoring`